### DISCUSSION TITLE
Manual ???

### DISCUSSION THREAD

**Discussion by hstriepe** [bitgapp/eqMac#532] [Q&A]
The FAQ does not tell you much about the features and how to actually use them apart from the basics - especially the Pro verison. Is there a more detailed manual anywhwere?

**Comment by hstriepe** [comment]
For example, with Pro Features enabled, how do I record app audio while the speaker is turned off?

**Reply by marksou** [reply]
Also happy to help. You could use an open source docs tool like Docusaurus.

**Comment by nodeful** [comment]
Good point, but I haven't had time to write those manuals just yet. I spend 90% if my time writing code and 10% replying to messages. Manual is definitely something I need to implement at some point, I thought to make a 5-10 minute YouTube video going over all the functionality. To answer your question about recording:
This feature is on the Roadmap and should be implemented this summer, but you can kinda get it to work using this method:
1. Download BlackHole: https://github.com/ExistentialAudio/BlackHole
2. Create an Aggregate device (BlackHole + your Output device) in Audio MIDI Setup.app
3. Select your aggregate device with eqMac
4. Use QuickTime player to record of BlackHole device

**Reply by hstriepe** [reply]
Is there a way to target a particular app as input with this routing?

**Comment by blcarson** [comment]
I'd be happy to contribute to this - it's honestly the only thing keeping me from upgrading to pro.

**Reply by nodeful** [reply]
Let's start thinking about this when I release v1.3.0 because that's when most features will be set in stone. I will create a YouTube video first and foremost about how to use eqMac and all the hidden features, after which we can focus on making a more detailed readable Manual :)
